config/i18n/all-langs.ts

```diff
@@ -185,8 +185,22 @@ export const i18nextCodes = {
   arabic: 'ar'
 };
 
-// These are for the language selector dropdown menu in the footer
 /* eslint-disable @typescript-eslint/naming-convention */
+// These are for language attribute direction in the html element
+export enum LangDirection {
+  english = 'ltr',
+  espanol = 'ltr',
+  chinese = 'ltr',
+  'chinese-traditional' = 'ltr',
+  italian = 'ltr',
+  portuguese = 'ltr',
+  ukrainian = 'ltr',
+  japanese = 'ltr',
+  german = 'ltr',
+  arabic = 'rtl'
+}
```

Comment:
```suggestion
/**
 * This array contains languages that use the RTL layouts.
 */
export const ltrLangs = ['arabic', 'urdu'];
```

Change this to a more straightforward array, and in the UI component, update your logic to default to LTR and override with RTL only for locales that match the included languages in the said array.

Reply:
I have spent a while reading the standard, to see if they say "don't explicitly add `dir='ltr'`, or do add it". But there are points mentioning that some elements have default `dir` like `dir='auto'`, and `dir='lft'`.

So I didn't add the `dir='ltr'`, because it may conflict with these elements. To be honest, I don't know if they will conflict with them, because of my lack of experience. What do you think should we explicitly add them, or keep the same logic

```ts
{...(isRtlLanguage && rtlDirectionAttribute)}
```